### PR TITLE
specify node version

### PR DESCRIPTION
**Pull request by hobbitronics**
add "engines" to package.json and run npm install

### CODE REVIEW

**Review by mtompset — COMMENTED** [review]
Why npm 8.15? The latest is 8.19.2 (https://www.npmjs.com/package/npm)
I'm assuming 16.17 was chosen because it is the LTS version (https://nodejs.org/en/)?

**Comment by hobbitronics** [comment]
> Why npm 8.15? The latest is 8.19.2 (https://www.npmjs.com/package/npm) I'm assuming 16.17 was chosen because it is the LTS version (https://nodejs.org/en/)?

that was the version that came with node 16.17.0, but I don't think we need to specify. I am going to try it without the npm version.

File: package.json [hunk 1]
```diff
@@ -1,5 +1,8 @@
 {
   "name": "idp-profile-ui",
+  "engines": {
+    "node": "=16.17.0"
```
Comment (forevermatt) [inline package.json:4]: What would you think of something like this instead, to allow non-breaking newer versions?
```suggestion
    "node": ">=16.0.0 <17.0.0"
```

**Comment by forevermatt** [comment]
@hobbitronics It looks like you'll want to run `npm install` again (or something) to get the node engines block in the package-lock.json file updated to use the range a well. Otherwise, this looks good to me. 👍

**Review by forevermatt — APPROVED** [review]
Approved, with one additional request.

**Pull request merged**